components/scream/src/physics/shoc/shoc_diag_second_moments_ubycond_impl.hpp

```diff
@@ -0,0 +1,57 @@
+
+#pragma once
+// Code placed below is included only once per translation unit
+
+#include "shoc_functions.hpp" // for ETI only but harmless for GPU
+#include "physics_functions.hpp" // also for ETI not on GPUs
+
+namespace scream {
+namespace shoc {
+
+template<typename S, typename D>
+KOKKOS_FUNCTION
+void Functions<S,D>
+::shoc_diag_second_moments_ubycond(
+    const MemberType& team, const Int& shcol, const Int& num_tracer,
+    const view_1d<Spack>& thl_sec, const view_1d<Spack>& qw_sec, const view_1d<Spack>& wthl_sec, const view_1d<Spack>& wqw_sec,
+    const view_1d<Spack>& qwthl_sec, const view_1d<Spack>& uw_sec, const view_1d<Spack>& vw_sec,const view_1d<Spack>& wtke_sec, 
+    const view_2d<Spack>& wtracer_sec)
+{
+  // Purpose of this subroutine is to diagnose the upper
+  //  boundary condition for the second order moments
+  //  needed for the SHOC parameterization.  Currently
+  //  set all to zero.
```

Comment:
Is this the current implementation for the boundary condition of the 2nd-order moments, or just a placeholder?

Reply:
It sounds to me this is just placeholder